### PR TITLE
Pioneer 16b glider deployment updates

### PR DESCRIPTION
**Pull request by cdobs**
Added P16b cruise to CruiseInformation.csv (QLR 3204-01607)

Added new G3 glider 913 to appropriate cross-mio files w/ new naming convention

Updated sensor bulkload with 2 new NBOSI CTDS, and updated manufacturer descriptions for existing NBOSI CTDS SN424 & 425

Updated sensor bulkload with new glider sensors:
ADCPAM (6)73245
DOSTAM 489 & 909
CTDGVM 9690
FLORTM 6642
PARADM 50333

### CODE REVIEW

File: bulk/node_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -108,6 +108,7 @@ CGVEH-GLDRCP-00514,,Node,1,Coastal Glider 514,COASTAL GLIDER 1000M,Teledyne Webb
 CGVEH-GLDRCP-00559,,Node,1,Coastal Glider 559,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,559,,,,,Woods Hole Oceanographic Institution
 CGVEH-GLDRCP-00564,,Node,1,Coastal Profiling Glider 564,COASTAL GLIDER 200M,Teledyne Webb,ASSY 302392-3,564,,,,,Woods Hole Oceanographic Institution
 CGVEH-GLDRCP-00583,,Node,1,Coastal Profiling Glider 583,COASTAL GLIDER 200M,Teledyne Webb,ASSY 302392-3,583,,,,,Woods Hole Oceanographic Institution
+CGVEH-GLDRCP-00913,,Node,1,Coastal Glider 913,COASTAL GLIDER 1000M,Teledyne Webb,ASSY GLD-0089-D,913,,,,,Woods Hole Oceanographic Institution
```
Comment (rtravis-whoi) [inline bulk/node_bulk_load-AssetRecord.csv:111]: updated model # looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -443,6 +443,7 @@ CGINS-ADCPAM-55980,,Sensor,1,ADCP Velocity Profiler (short range) for Glider: AD
 CGINS-ADCPAM-62598,N00131,Sensor,1,ADCP Velocity Profiler (short range) for Glider: ADCPA Series M,ADCP for Slocum Glider,Teledyne RDI,Explorer DVL 600 kHz,62598,,20130301,,,
 CGINS-ADCPAM-72777,,Sensor,1,ADCP Velocity Profiler (short range) for Glider: ADCPA Series M,ADCP for Slocum Glider,Teledyne RDI,Explorer DVL 600 kHz,672777,,,,,
 CGINS-ADCPAM-73107,,Sensor,1,ADCP Velocity Profiler (short range) for Glider: ADCPA Series M,ADCP for Slocum Glider,Teledyne RDI,Explorer DVL 600 kHz,673107,,,,,
+CGINS-ADCPAM-73245,,Sensor,1,ADCP Velocity Profiler (short range) for Glider: ADCPA Series M,ADCP for Slocum Glider,Teledyne RDI,Explorer DVL 600 kHz,673245,,,,,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:446]: looks good (for new G3 glider 913)

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -530,8 +531,10 @@ CGINS-CAMDSA-00122,,Sensor,0,Underwater Digital Still Camera: CAMDS Series A,CAM
...
 CGINS-CTDAVN-00104,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Sea-Bird Electronics,SBE AUV Payload CTD,104,,,,CGSN AUV Instrument Pool,
-CGINS-CTDAVN-00424,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Sea-Bird Electronics,SBE AUV Payload CTD,424,,,,CGSN AUV Instrument Pool,
-CGINS-CTDAVN-00425,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Sea-Bird Electronics,SBE AUV Payload CTD,425,,,,CGSN AUV Instrument Pool,
+CGINS-CTDAVN-00424,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Neil Brown Ocean Sensors Inc,NBOSI AUV Payload CTD,424,,,,CGSN AUV Instrument Pool,
+CGINS-CTDAVN-00425,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Neil Brown Ocean Sensors Inc,NBOSI AUV Payload CTD,425,,,,CGSN AUV Instrument Pool,
+CGINS-CTDAVN-00526,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Neil Brown Ocean Sensors Inc,NBOSI AUV Payload CTD,526,,,,CGSN AUV Instrument Pool,
+CGINS-CTDAVN-00527,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Neil Brown Ocean Sensors Inc,NBOSI AUV Payload CTD,527,,,,CGSN AUV Instrument Pool,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:537]: updates to 424 and 425 look good, and 526 and 527 new additions look good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 4]
```diff
@@ -686,6 +689,7 @@ CGINS-CTDGVM-09363,,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Slocum Gli
 CGINS-CTDGVM-09364,,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Slocum Glider,Sea-Bird Electronics,SBE Slocum Glider Payload CTD (GP-CTD),9364,,,,CGSN Glider Instrument Pool,
 CGINS-CTDGVM-09453,,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Slocum Glider,Sea-Bird Electronics,SBE Slocum Glider Payload CTD (GP-CTD),9453,,,,CGSN Glider Instrument Pool,Woods Hole Oceanographic Institution
 CGINS-CTDGVM-09658,,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Slocum Glider,Sea-Bird Electronics,SBE Slocum Glider Payload CTD (GP-CTD),9658,,,,,
+CGINS-CTDGVM-09690,,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Slocum Glider,Sea-Bird Electronics,SBE Slocum Glider Payload CTD (GP-CTD),9690,,,,,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:692]: looks good - vendor doc not yet posted to vault but I have copy of packet & able to verify (for new G3 glider 913)

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 5]
```diff
@@ -1220,6 +1224,7 @@ CGINS-DOSTAM-00436,OL000609,Sensor,1,Dissolved Oxygen Stable Response for Glider
 CGINS-DOSTAM-00437,OL000169,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,437,,20121227,25000,CGSN Glider Instrument Pool,
 CGINS-DOSTAM-00438,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,438,,,,CGSN Glider Instrument Pool,
 CGINS-DOSTAM-00480,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,480,,,,CGSN Glider Instrument Pool,
+CGINS-DOSTAM-00489,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,489,,,,CGSN Glider Instrument Pool,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:1227]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 6]
```diff
@@ -1237,6 +1242,7 @@ CGINS-DOSTAM-00604,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA
 CGINS-DOSTAM-00605,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,605,,,,CGSN Glider Instrument Pool,
 CGINS-DOSTAM-00662,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,662,,,,CGSN Glider Instrument Pool,Woods Hole Oceanographic Institution
 CGINS-DOSTAM-00864,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,864,,,,,
+CGINS-DOSTAM-00909,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,909,,,,,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:1245]: looks good (for new G3 glider 913)

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 7]
```diff
@@ -1488,6 +1494,7 @@ CGINS-FLORTM-04472,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M
 CGINS-FLORTM-04473,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,4473,,,,CGSN Glider Instrument Pool,
 CGINS-FLORTM-05133,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,5133,,,,CGSN Glider Instrument Pool,Woods Hole Oceanographic Institution
 CGINS-FLORTM-06168,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLC,6168,,,,,
+CGINS-FLORTM-06642,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLC,6642,,,,,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:1497]: looks good (for new G3 glider 913)

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 8]
```diff
@@ -2108,6 +2115,7 @@ CGINS-PARADM-50277,,Sensor,1,Photosynthetically Available Radiation for Glider:
 CGINS-PARADM-50304,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50304,,,,CGSN Glider Instrument Pool,
 CGINS-PARADM-50319,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50319,1.2,,,,
 CGINS-PARADM-50324,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50324,,,,CGSN Glider Instrument Pool,
+CGINS-PARADM-50333,,Sensor,1,Photosynthetically Available Radiation for Glider: PARAD Series M,PARAD for Slocum Glider,Biospherical Instruments,QSP-2155,50333,,,,CGSN Glider Instrument Pool,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:2118]: looks good (for new G3 glider 913)

File: cruise/CruiseInformation.csv [hunk 9]
```diff
@@ -245,3 +245,4 @@ AR52,Neil Armstrong,2021-03-29T20:19:00,2021-04-17T11:37:00,Pioneer 16
 TS20210419,Tenacious II,2021-04-19T05:00:00,2021-04-20T04:00:00,Pioneer 16a glider recoveries
 EK20210419,Elakha,2021-04-19T14:50:00,2021-04-19T20:38:00,Endurance CE01ISSP-00017 and CE05MOAS-GL383-00010
 EK20210429,Elakha,2021-04-29T15:25:00,2021-04-29T22:17:00,Endurance CSPP and glider operations
+TS20210517,Tenacious II,2021-05-17T11:00:00,2021-05-18T11:00:00,Pioneer 16b glider deployments
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:248]: looks good - verified via QLR

File: vocab/vocab.csv [hunk 10]
```diff
@@ -792,6 +792,13 @@ CP05MOAS-GL559-02-FLORTM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,3-W
 CP05MOAS-GL559-03-CTDGVM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
 CP05MOAS-GL559-04-DOSTAM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
 CP05MOAS-GL559-05-PARADM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CP05MOAS-G0913,Coastal Pioneer,Mobile Assets,Coastal Glider 913,,,,0,1000
+CP05MOAS-G0913-00-ENG000000,Coastal Pioneer,Mobile Assets,Coastal Glider 913,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:796]: this needs to be updated to 'G3 Slocum Gliders'

File: vocab/vocab.csv [hunk 11]
```diff
@@ -792,6 +792,13 @@ CP05MOAS-GL559-02-FLORTM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,3-W
...
+CP05MOAS-G0913-01-ADCPAM000,Coastal Pioneer,Mobile Assets,Coastal Glider 913,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,1000
+CP05MOAS-G0913-02-FLORTM000,Coastal Pioneer,Mobile Assets,Coastal Glider 913,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,1000
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:798]: this should be ECO Puck FLBBCD-SLC,  not -SLK

File: vocab/vocab.csv [hunk 12]
```diff
@@ -792,6 +792,13 @@ CP05MOAS-GL559-02-FLORTM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,3-W
 CP05MOAS-GL559-03-CTDGVM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
 CP05MOAS-GL559-04-DOSTAM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
 CP05MOAS-GL559-05-PARADM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CP05MOAS-G0913,Coastal Pioneer,Mobile Assets,Coastal Glider 913,,,,0,1000
+CP05MOAS-G0913-00-ENG000000,Coastal Pioneer,Mobile Assets,Coastal Glider 913,Mobile Asset Controller,Teledyne Webb,G3 Slocum Gliders,0,1000
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:796]: update looks good - changed fr G2 to G3

File: vocab/vocab.csv [hunk 13]
```diff
@@ -792,6 +792,13 @@ CP05MOAS-GL559-02-FLORTM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,3-W
...
+CP05MOAS-G0913-01-ADCPAM000,Coastal Pioneer,Mobile Assets,Coastal Glider 913,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,1000
+CP05MOAS-G0913-02-FLORTM000,Coastal Pioneer,Mobile Assets,Coastal Glider 913,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLC,0,1000
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:798]: FLORTM correction looks good - FLBBCD-SLK changed to -SLC

**Review by rtravis-whoi — APPROVED** [review]
updates look good; all sensors for P16b glider deployments are either already in sensor_bulk_load or added via this PR.

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 14]
```diff
@@ -180,6 +180,7 @@ CGCON-GLDRCP-00514,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL514 Co
 CGCON-GLDRCP-00559,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL559 Controller,Teledyne Webb,G2 Slocum Gliders,559,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00564,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,564,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00583,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL583 Controller,Teledyne Webb,G2 Slocum Gliders,583,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
+CGCON-GLDRCP-00913,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL913 Controller,Teledyne Webb,G3 Slocum Gliders,913,,,,,Woods Hole Oceanographic Institution
```
[the remark this answers is not in the document]
Comment (snwhite) [reply bulk/eng_bulk_load-AssetRecord.csv:183]: Looks good.

File: bulk/node_bulk_load-AssetRecord.csv (hunk 1 above)
Comment (snwhite) [reply bulk/node_bulk_load-AssetRecord.csv:111]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 4 above)
Comment (snwhite) [reply bulk/sensor_bulk_load-AssetRecord.csv:692]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 6 above)
Comment (snwhite) [reply bulk/sensor_bulk_load-AssetRecord.csv:1245]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 7 above)
Comment (snwhite) [reply bulk/sensor_bulk_load-AssetRecord.csv:1497]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 8 above)
Comment (snwhite) [reply bulk/sensor_bulk_load-AssetRecord.csv:2118]: Looks good.

File: cruise/CruiseInformation.csv (hunk 9 above)
Comment (snwhite) [reply cruise/CruiseInformation.csv:248]: Looks good.

File: vocab/vocab.csv [hunk 15]
```diff
@@ -792,6 +792,13 @@ CP05MOAS-GL559-02-FLORTM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,3-W
...
+CP05MOAS-G0913-03-CTDGVM000,Coastal Pioneer,Mobile Assets,Coastal Glider 913,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+CP05MOAS-G0913-04-DOSTAM000,Coastal Pioneer,Mobile Assets,Coastal Glider 913,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CP05MOAS-G0913-05-PARADM000,Coastal Pioneer,Mobile Assets,Coastal Glider 913,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
```
Comment (snwhite) [inline vocab/vocab.csv:801]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 5 above)
Comment (snwhite) [reply bulk/sensor_bulk_load-AssetRecord.csv:1227]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 3 above)
Comment (snwhite) [reply bulk/sensor_bulk_load-AssetRecord.csv:537]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 2 above)
Comment (snwhite) [reply bulk/sensor_bulk_load-AssetRecord.csv:446]: Looks good.

**Review by snwhite — APPROVED** [review]
Looks good.

**Pull request merged**